pontoon/api/tests/test_schema.py

```diff
@@ -0,0 +1,149 @@
+from django.test import TestCase
+from django.utils.encoding import force_text
+
+
+class TestCyclicQueries(TestCase):
+
+    def test_projects(self):
+        body = {
+            'query': '''{
+                projects {
+                    name
+                }
+            }'''
+        }
+
+        response = self.client.get('/graphql', body,
+                                   HTTP_ACCEPT="application/json")
+
+        self.assertEqual(response.status_code, 200)
+        self.assertJSONEqual(
+            force_text(response.content),
+            {"data": {"projects": [
+                {"name": "Pontoon Intro"}]}})
+
+    def test_projects_localizations(self):
+        body = {
+            'query': '''{
+                projects {
+                    localizations {
+                        locale {
+                            name
+                        }
+                    }
+                }
+            }'''
+        }
+
+        response = self.client.get('/graphql', body,
+                                   HTTP_ACCEPT="application/json")
+
+        self.assertEqual(response.status_code, 200)
+        self.assertJSONEqual(
+            force_text(response.content),
```

Comment:
Why do you use ``force_text``?

Reply:
I ran into some encoding problem when I was writing these tests, but now I just removed `force_text` and everything works fine.